packages/blocks/src/api/raw-handling/shortcode-converter.js

```diff
@@ -92,8 +92,15 @@ function segmentHTMLToShortcodeBlock(
 			( schema ) => schema.shortcode( match.shortcode.attrs, match )
 		);
 
+		const blockType = getBlockType( transformation.blockName );
+		if ( ! blockType ) {
+			return [
+				...segmentHTMLToShortcodeBlock( beforeHTML ),
+				...segmentHTMLToShortcodeBlock( afterHTML ),
+			];
+		}
```

Comment:
Do you mind moving this change into a separate PR, I think we'll want a unit test here to make sure we're handling this case properly.

Reply:
Ok sure, I will remove these changes from this PR and will create a separate PR. It might take some time for me to come up with the PR. I will be writing front-end/javascript test cases for the first time. Need some upskilling 😄